frozendict/__init__.py

```diff
@@ -18,31 +18,35 @@ class frozendict(collections.Mapping):
     An immutable wrapper around dictionaries that implements the complete :py:class:`collections.Mapping`
     interface. It can be used as a drop-in replacement for dictionaries where immutability is desired.
     """
-
+    
     dict_cls = dict
-
+    
+    @classmethod
+    def fromkeys(cls, seq, value=None, *args, **kwargs):
+        return cls(dict.fromkeys(seq, value, *args, **kwargs))
+    
     def __init__(self, *args, **kwargs):
         self._dict = self.dict_cls(*args, **kwargs)
         self._hash = None
-
+    
```

Comment:
Why all these extra whitespace additions? :confused:

Reply:
It's my style.